security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeUtils.php

```diff
@@ -4,7 +4,11 @@
  * Copyright (C) 2017-2020 Frank Wall
  * Copyright (C) 2015 Deciso B.V.
  * Copyright (C) 2010 Jim Pingle <jimp@pfsense.org>
+ * Copyright (C) 2010 Ermal Luçi
  * Copyright (C) 2008 Shrew Soft Inc. <mgrooms@shrew.net>
+ * Copyright (C) 2005-2006 Colin Smith <ethethlay@gmail.com>
+ * Copyright (C) 2004-2007 Scott Ullrich <sullrich@gmail.com>
+ * Copyright (C) 2003-2004 Manuel Kasper <mk@neon1.net>
```

Comment:
oh no what are you doing :D

ffc170354 334aa32f0f71417 I wrote this ;)

Reply:
I was unable to identify who wrote this in the first place. Thanks for letting me know, I'll revert the copyright change.

Reply:
Fixed in https://github.com/opnsense/plugins/pull/2055/commits/be4e8c4bc284ce8546d3e3dd07c235bf9f1993f4.